src/PrestaShopBundle/Controller/Admin/Sell/Order/CartController.php

```diff
@@ -283,4 +336,34 @@ private function getErrorMessages(Exception $e)
             CartRuleValidityException::class => $e->getMessage(),
         ];
     }
+
+    /**
+     * @param Request $request
+     * @param int $cartId
+     *
+     * @return UpdateProductQuantityInCartCommand
+     */
+    private function getAddProductToCartCommand(Request $request, int $cartId): UpdateProductQuantityInCartCommand
+    {
+        $productId = $request->request->getInt('product_id');
+        $quantity = $request->request->getInt('quantity');
+        $combinationId = $request->request->getInt('combination_id');
+
+        if ($customizations = $request->request->get('customization')) {
```

Comment:
The request parameter should `customizations` if it's an array

Reply:
Also you shouldn't execute a command in this method called `getAddProductToCartCommand`, the method should only create the object an not perform an action at the same time (which is not explicit in the method name)

You could rename the method into something like `getAddProductToCartCommandAndCreateOrUpdateCustomizations` but it would be clearer to have a dedicated method for this `updateProductCostomizations(Request $request, int $cartId)`

Reply:
> The request parameter should `customizations` if it's an array

It comes as customization[1] customization[2] because of html input name. You think its ok to name the single input in plural form like `<input name=customizations[1]>` ?

> Also you shouldn't execute a command in this method called getAddProductToCartCommand

yes I agree, i should have left a todo here, It was just a quick draft, as cusomizations should be done in even separate pr maybe.

Reply:
Yes I think `<input name=customizations[1]>` is perfectly fine you see straight away that it contains an array

Reply:
I'm not sure to understand 😅